plugins/community/timostamm-protobuf-ts/v2.8.2/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM node:19.3.0-alpine3.17 AS build
```

Comment:
Should we stick to a node LTS version instead of a dev version?

Reply:
Ye we probably should, let me change that to be consistent.